benches/common/mod.rs

```diff
@@ -0,0 +1,33 @@
+use std::collections::HashMap;
+
+use anyhow::{bail, Result};
+use camino::Utf8PathBuf;
+use lurk::cli::paths::lurk_default_dir;
+use lurk::config::lurk_config;
+use once_cell::sync::Lazy;
+
+/// Temporary cache location that persists until reboot or other cleanup process,
+/// so the user can run multiple benchmarks without regenerating params each time
+pub const PUBLIC_PARAMS_BENCH_PATH: &str = "/var/tmp/lurk_benches/public_params";
```

Comment:
I'd vote for this to live in the `.lurk` directory. Maybe we can use an inner `dev` folder for that.

This is again a subjective thing, but I don't like having apps that populate folders out of my `$HOME` when it's not needed. It's easier for my SSD to just be occupied with useless data that way.

Reply:
That makes sense to me. Note that `fcomm::data_dir()` is also located in `/var/tmp`, but perhaps not worth changing anymore

Reply:
I believe pq's general take is that all the public params should just live in the same directory. Also, this is less of a concern, but params are very large and any dedupilcation would be good.

See: https://zulip.lurk-lab.com/#narrow/stream/44-compilers/topic/Parallel.20witness.20generation/near/121174

Reply:
Removed in favor of the default param location, but left the `set_bench_config` function in case the user wants to change it or the perf config as before.